subprojects/language-native/src/integTest/groovy/org/gradle/language/swift/tasks/UnexportMainSymbolIntegrationTest.groovy

```diff
@@ -35,10 +38,32 @@ class UnexportMainSymbolIntegrationTest extends AbstractInstalledToolChainIntegr
         """
     }
 
+    @Issue("https://github.com/gradle/gradle-native/issues/304")
+    def "clean build works"() {
+        writeMainSwift()
+
+        expect:
+        succeeds("clean", "unexport", "assemble")
+    }
+
+    @NotYetImplemented
+    @Issue("https://github.com/gradle/gradle-native/issues/297")
+    def "unexport is incremental"() {
```

Comment:
Should we make a separate PR for this fix? I don't think we should merge code that adds NotYetImplmented. What do you think?

Reply:
I thought it might have been fixed by the same change, but it wasn't. There's something deeper/weirder going on.  This looked like a bug we would fix soon-ish anyways, so I thought it was worth keeping the test.